style.css

```diff
@@ -0,0 +1,171 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&family=Poppins:ital@0;1&display=swap');
+
+html,
+body {
+  padding: 1em;
+  margin: 0;
+  font-family: Lato, Poppins, sans-serif;
+  box-sizing: border-box;
+  color: #333;
+}
+
+header {
+  position: fixed;
+  right: 0;
+  left: 0;
+  margin: 10px 10%;
+}
+
+#navbar {
+  border: 3px solid;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px;
+  height: 30px;
+}
+
+ul {
+  list-style: none;
+}
+
+#navbar ul {
+  display: flex;
+  list-style: none;
+  gap: 10px;
+  align-items: center;
+}
+
+.active {
```

Comment:
- [ ]  Good work writing nice style rules. My suggestion is you should avoid the use of `!important` and instead start out by writing general style rules and become more specific as you progress. Using `important` can present debugging challenges 😃